core/common/src/main/java/alluxio/wire/MasterInfo.java

```diff
@@ -63,16 +80,61 @@ public long getLastUpdatedTimeMs() {
     return mLastUpdatedTimeMs;
   }
 
+  /**
+   * @param address the master address information
+   * @return the master information
+   */
+  public MasterInfo setAddress(Address address) {
+    mAddress = address;
+    return this;
+  }
+
+  /**
+   * @param id the master id
+   * @return the master information
+   */
+  public MasterInfo setId(long id) {
+    mId = id;
+    return this;
+  }
+
+  /**
+   * @param lastUpdatedTimeMs the last update time in ms
+   * @return the master information
+   */
+  public MasterInfo setLastUpdatedTimeMs(long lastUpdatedTimeMs) {
+    mLastUpdatedTimeMs = lastUpdatedTimeMs;
+    return this;
+  }
+
   @Override
   public String toString() {
     return MoreObjects.toStringHelper(this).add("id", mId).add("address", mAddress)
-        .add("lastUpdatedTimeMs", mLastUpdatedTimeMs).toString();
+            .add("lastUpdatedTimeMs", mLastUpdatedTimeMs).toString();
   }
 
   /**
-   * Updates the last updated time of the master in ms.
+   * Updates the last updated time of the master (in milliseconds).
```

Comment:
ms = milliseconds, so do you think this change is not necessary?

Reply:
Yes, but there is another test failed by accident, I use this commit to re-run it.